tonic/src/status.rs

```diff
@@ -546,6 +582,13 @@ impl fmt::Debug for Status {
             builder.field("metadata", &self.metadata);
         }
 
+        #[cfg(feature = "transport")]
+        {
+            if self.h2_error.is_some() {
+                builder.field("h2_error", &self.h2_error);
+            }
```

Comment:
```suggestion
            builder.field("h2_error", &self.h2_error);
```

I guess staying close to the actual struct definition is fine as well.

Reply:
Made this change as well.